mongomock/collection.py

```diff
@@ -1146,7 +1146,7 @@ def delete_one(self, filter, session=None):
         validate_is_mapping('filter', filter)
         return DeleteResult(self._delete(filter), True)
 
-    def delete_many(self, filter, session=None):
+    def delete_many(self, filter, collation=None, session=None):
         validate_is_mapping('filter', filter)
         return DeleteResult(self._delete(filter, multi=True), True)
```

Comment:
According to our guidelines, we don't want to have features silently ignored: if `mongomock` does not handle a case it should says so explicitly. Here that would mean to raise a `NotImplementedError` indicating that collation is a valid keyword but that Mongomock does not handle it yet. Of course you should raise that exception only if the collation parameter would imply a different behaviour (i.e. it's not None).

Note that this was not done for the `session` parameter which is a bug.